schema/artist/statuses.js

```diff
@@ -10,61 +10,72 @@ const ArtistStatusesType = new GraphQLObjectType({
   fields: {
     artworks: {
       type: GraphQLBoolean,
+      resolve: ({ published_artworks_count }) => !!published_artworks_count,
+    },
+    shows: {
+      type: GraphQLBoolean,
       resolve: ({ id }) => {
-        return gravity(`artist/${id}/artworks`, {
-          published: true,
-          size: 1,
-        }).then(artworks => !!artworks.length);
+        return !!total(`related/shows`, {
+          artist_id: id,
+          displayable: true,
+          size: 0,
+        });
```

Comment:
I think this is just going to always be `true` since you're just bool casting the Promise.

`total` returns a Promise which resolves an Integer count:

```javascript
total(...).then(n => n > 0)
```